tests/testsuite/net_config.rs

```diff
@@ -67,10 +67,11 @@ fn net_retry_git_outputs_warning() {
 
     p.cargo("check -v -j 1")
         .with_status(101)
-        .with_stderr_contains(
-            "[WARNING] spurious network error \
-             (2 tries remaining): [..]",
-        )
-        .with_stderr_contains("[WARNING] spurious network error (1 tries remaining): [..]")
+        .with_stderr_data(str![[r#"
+...
+[WARNING] spurious network error (2 tries remaining): [7] Couldn't connect to server (Failed to connect to 127.0.0.1 port 11 after [..] ms: Couldn't connect to server)[..]
```

Comment:
Is the port number fixed?

Reply:
Based on my local tests, I assume it is.  However, I'll redact it if it fails in CI.